app/common/state/tabState.js

```diff
@@ -610,7 +610,7 @@ const tabState = {
     state = tabState.removeTabField(state, 'messageBoxDetail')
     state = tabState.removeTabField(state, 'frame')
     state = state.delete('tabsInternal')
-    return state.delete('tabs')
+    return state.set('tabs', Immutable.List())
```

Comment:
I think that might be a problem because at some point in the near future we won't be deleting the tabs and so this won't be equivalent to a default state